frontend/src/components/ShootWorkers/WorkerInputGeneric.vue

```diff
@@ -314,15 +314,27 @@ export default {
         }
       }
     },
+    selectedZones: {
+      get: function () {
+        return ensureUniqueSelectedItemValues(this.worker.zones)
```

Comment:
How can we have duplicate selected items?

Reply:
because of some shady yaml hackers ->discussed offline